.checkstyle

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is useless.
Please, remove it from your repo and add it to the .gitignore.